### PR TITLE
Add file 01-character-input.py

### PR DESCRIPTION
**Pull request by MaiBil**

### CODE REVIEW

File: practicepython.org/01-character-input.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/user/bin/env python
+
+# author: Mailen Bilsky
+# date: 02/10/2018
+
+# title: Character Input
+# url: https://www.practicepython.org/exercise/2014/01/29/01-character-input.html
+# time complexity: O(n), with n being the amount of numbers in the input string
+
+def get_name_and_age(name,age,number):
```
Comment (cdipietro) [inline practicepython.org/01-character-input.py]: According to Python's [PEP-8](https://www.python.org/dev/peps/pep-0008/) you need to put a single space after commas. 
Remember you can always check if your code is compliant to [PEP-8](https://www.python.org/dev/peps/pep-0008/) by running the analyzing your code with [`pycodestyle`](http://pycodestyle.pycqa.org/en/latest/) package.

File: practicepython.org/01-character-input.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    that tells them the year that they will turn 100 years old.
+
+	Extras:
+
+	Add on to the previous program by asking the user for another number and printing out that many copies of the 
+	previous message. (Hint: order of operations exists in Python)
+	Print out that many copies of the previous message on separate lines. (Hint: the string "\n is the same as 
+	pressing the ENTER button)
+    """
+    year = 2018 + (100 - int(age))
```
Comment (cdipietro) [inline practicepython.org/01-character-input.py]: What would happen when someone gets to run this program in the year 2020? Would the answer still be correct? Maybe you need to find a way to figure out which year we are in programmatically and build upon that.

File: practicepython.org/01-character-input.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+#!/user/bin/env python
+
+# author: Mailen Bilsky
+# date: 02/10/2018
+
+# title: Character Input
+# url: https://www.practicepython.org/exercise/2014/01/29/01-character-input.html
+# time complexity: O(n), with n being the amount of numbers in the input string
```
Comment (cdipietro) [inline practicepython.org/01-character-input.py]: Why is this code O(n)? Could you elaborate on that?

File: practicepython.org/01-character-input.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+	Print out that many copies of the previous message on separate lines. (Hint: the string "\n is the same as 
+	pressing the ENTER button)
+    """
+    year = 2018 + (100 - int(age))
+    print(("You will be 100 years old on the year %s. \n" % year)*int(number))
+
+name = input("What's your name?: ")
+age = input("How old are you?: ")
+number = input("Give me another number: ")
+get_name_and_age(name,age)
```
Comment (cdipietro) [inline practicepython.org/01-character-input.py]: Remember to add a new line at the end of each file. Also, please check your code with `pycodestyle` and fix all the violations to the coding style guide.

File: practicepython.org/01-character-input.py [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+	"""
+    Create a program that asks the user to enter their name and their age. Print out a message addressed to them 
+    that tells them the year that they will turn 100 years old.
+
+	Extras:
+
+	Add on to the previous program by asking the user for another number and printing out that many copies of the 
+	previous message. (Hint: order of operations exists in Python)
+	Print out that many copies of the previous message on separate lines. (Hint: the string "\n is the same as 
+	pressing the ENTER button)
```
Comment (cdipietro) [inline practicepython.org/01-character-input.py]: Do you plan to do the extra part of this challenge as well?

Comment (MaiBil) [reply practicepython.org/01-character-input.py]: I did! It's on the code. On line 23 when I multiply the string by int(number)

Comment (cdipietro) [reply practicepython.org/01-character-input.py]: > Add on to the previous program by asking the user for **another** number and printing out that many copies of the previous message.

**Comment by MaiBil** [comment]
> Much better! One last remaining change: make sure you always add a blank line at the end of the file.
> Do that last thing, and you'll get your first PR approved 😉

Im pretty sure it alredy has a last line. It says it has 34 lines, even though when I go to Files changed -> View, it only shows me 33 lines (and says 34 lines (24 sloc)  1.12 KB).

I will anyway commit the file again, with the 34th line (that was alredy there!).

**Comment by MaiBil** [comment]
> Much better! One last remaining change: make sure you always add a blank line at the end of the file.
> Do that last thing, and you'll get your first PR approved 😉

Im pretty sure it alredy has a last line. It says it has 34 lines, even though when I go to Files changed -> View, it only shows me 33 lines (and says 34 lines (24 sloc)  1.12 KB).

I tried to add the file again (the one that alredy has the 34th line) but it doesn't let me (because it has no changes!)

**Comment by MaiBil** [comment]
> Much better! One last remaining change: make sure you always add a blank line at the end of the file.
> Do that last thing, and you'll get your first PR approved 😉

Im pretty sure it alredy has a last line. It says it has 34 lines, even though when I go to Files changed -> View, it only shows me 33 lines (and says 34 lines (24 sloc)  1.12 KB).

I tried to add the file again (the one that alredy has the 34th line) but it doesn't let me (because it has no changes!)

**Comment by MaiBil** [comment]
> Much better! One last remaining change: make sure you always add a blank line at the end of the file.
> Do that last thing, and you'll get your first PR approved 😉

Im pretty sure it alredy has a last line. It says it has 34 lines, even though when I go to Files changed -> View, it only shows me 33 lines (and says 34 lines (24 sloc)  1.12 KB).

I tried to add the file again (the one that alredy has the 34th line) but it doesn't let me (because it has no changes!)

**Comment by MaiBil** [comment]
> Much better! One last remaining change: make sure you always add a blank line at the end of the file.
> Do that last thing, and you'll get your first PR approved 😉

Im pretty sure it alredy has a last line. It says it has 34 lines, even though when I go to Files changed -> View, it only shows me 33 lines (and says 34 lines (24 sloc)  1.12 KB).

I tried to add the file again (the one that alredy has the 34th line) but it doesn't let me (because it has no changes!)

**Comment by MaiBil** [comment]
> Much better! One last remaining change: make sure you always add a blank line at the end of the file.
> Do that last thing, and you'll get your first PR approved 😉

Im pretty sure it alredy has a last line. It says it has 34 lines, even though when I go to Files changed -> View, it only shows me 33 lines (and says 34 lines (24 sloc)  1.12 KB).

I tried to add the file again (the one that alredy has the 34th line) but it doesn't let me (because it has no changes!)

**Comment by MaiBil** [comment]
Ummm.... It says I "closed" this. I have no idea what I did. How can I reverse it??

**Pull request merged**